homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
+    const container = document.getElementById('root');
+    fetchJSON(url, (err, repo) => {
+      if (err) {
+        createAndAppend('div', container, { text: err.message, class: 'alert-error' });
+      }
+      ul.innerHTML = '';
+
+      const li = createAndAppend('li', ul, { class: 'info-list' });
+      const table = createAndAppend('table', li);
+      const tbody = createAndAppend('tbody', table);
+      const trName = createAndAppend('tr', tbody);
```

Comment:
You are repeating yourself here four times: create a `tr` with two `td` children. Can you make your code DRYer by introducing a function called, say, `addRow` that you call four times, supplying the information it needs for each row?

For the first element, containing the `a` you need to do something extra. If you were to create an `addRow` function, you could do something like this:

```js
const firstRow = addRow(tbody, 'Name', '');
createAddAppend('a', firstRow.lastChild, { href: ..., text: .... });
```